docs/components/codeblock.md

```diff
@@ -217,6 +217,40 @@ Sets the display theme of the component.
 />
 ```
 
+### isSingleLine
```

Comment:
nit: Can we move this prop above the `theme` prop in the docs?

Reply:
Fixed